sumcheck/src/prover/high_degree.rs

```diff
@@ -398,6 +433,21 @@ fn sumcheck_round<E: FieldElement>(
                 let eq_at_zero = eq_ml.evaluations()[2 * i];
                 let eq_at_one = eq_ml.evaluations()[2 * i + 1];
 
+                let periodic_at_zero = periodic_table.get_periodic_values_at(2 * i);
+                let periodic_at_one = periodic_table.get_periodic_values_at(2 * i + 1);
+
+                // add evaluation of periodic columns
+                evals_zero
+                    .iter_mut()
+                    .skip(num_mls)
+                    .enumerate()
+                    .for_each(|(i, ev)| *ev = periodic_at_zero[i]);
+                evals_one
+                    .iter_mut()
+                    .skip(num_mls)
+                    .enumerate()
+                    .for_each(|(i, ev)| *ev = periodic_at_one[i]);
```

Comment:
If we change `get_periodic_values_at()` to write to a buffer, we may be able to do this as:
```rust
periodic_table.get_periodic_values_at(2 * i, &mut evals_zero[num_mls + 1..]);
periodic_table.get_periodic_values_at(2 * i + 1, &mut evals_one[num_mls + 1..]);
```

Reply:
Done